content/pages/index.html

```diff
@@ -343,14 +343,10 @@ <h3 id="heading">Create IEEE 829 or free text Test Plan</h3>
                 <img style="background-color: inherit" src="/images/conf/piterpy.png" alt="PiterPy logo" title="PiterPy St. Petersburg">
                 <img style="background-color: inherit" src="/images/conf/openfest.png" alt="OpenFest logo" title="Open Fest Sofia">
 
-                <div>
-                    <img style="background-color: inherit" src="/images/conf/fosdem.png" alt="FOSDEM'19 logo" title="FOSDEM Brussels">
-
-                    <div>
-                        <a href="#" title="Watch video" onclick="showVideo('9kOnf5YXquA')">
-                            <i class="fa fa-play fa-3x"></i>
-                        </a>
-                    </div>
+                <div class="fosdem-img" title="FOSDEM Brussels">
+                    <a href="#" title="Watch video" onclick="showVideo('9kOnf5YXquA')">
+                        <i class="fa fa-play fa-2x"></i>
```

Comment:
Remove `fa-2x`, let's use a smaller play button here. Larger ones will make the section look busier when you add more of them.